src/schema/objects/metadata.yaml

```diff
@@ -1073,6 +1119,26 @@ GeneratedBy:
           URI:
             type: string
             format: uri
+GeneticInfo:
```

Comment:
`GeneticInfo` is the full contents of `/genetic_info.json`. Is this metadata? Should this be somewhere else? I think it is still good to have it somewhere in `objects.` Could be `objects.documents`, including `dataset_description.json`.